src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -95,7 +95,7 @@ public Optional<String> getAssignment(
      * @return
      * @throws Exception
      */
-    public Optional<String> getAssignment(String subjectKey, String experimentKey) {
+    public Optional<String> getAssignment(String subjectKey, String experimentKey) throws Exception {
```

Comment:
I don't think we should have a checked exception as part of this method signature; it is more difficult for consumer, as they have to use a try/catch block to call our API.

If the method body can throw an exception, it should be RuntimeException or something that inherits from RuntimeException.

Reply:
Yeah make sense! Removed the throw Exception part